### PR TITLE
Initial branch

### PR DESCRIPTION
**Pull request by venkykatta**
Adding the requirements.txt file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,17 @@
...
   2.And also  setup the [virtualenv](https://docs.python.org/3/tutorial/venv.html) in your local directory.
 
-  3.And finally run the application with command as "python3 -m flask run"
\ No newline at end of file
+  3.And finally run the application with command as "python3 -m flask run"
+
+  ## How to Run:
+   1.Once a collaborator pulls down your project - they can then install a fresh python environment that is,
+
+      "python3 -m venv local_python_environment"
```
Comment (su66uu) [inline README.md:16]: You can use simpler env name like `venv` instead of ocal_python_environment

File: README.md [hunk 2]
```diff
@@ -8,4 +8,17 @@
...
+    
+    2.And then activate that environment and install from your requirements.txt which you have included in your version control:
+
+       "source local_python_environment/bin/activate"
```
Comment (su66uu) [inline README.md:20]: So here it will be just `source venv/bin/activate` instead of long command

File: README.md [hunk 3]
```diff
@@ -8,4 +8,17 @@
...
+
+    3.Then after we install  the requirements.txt file in our local directory that is like:
+    
+         "(local_python_environment) $ pip install -r requirements.txt"
```
Comment (su66uu) [inline README.md:24]: Why (local_python_environment)?

**Review by su66uu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (venkykatta) [reply README.md:16]: Yes, we use local_python_environment instead of 'venv'

File: README.md (hunk 2 above)
Comment (venkykatta) [reply README.md:20]: yes , that's correct

File: README.md (hunk 1 above)
Comment (venkykatta) [reply README.md:16]: sorry.., we use **venv** instead of local_python_environment

**Pull request merged**